lib/typhoeus/response/header.rb

```diff
@@ -47,7 +47,7 @@ def parse
       # @return [ void ]
       def process_line(header)
         key, value = header.split(':', 2)
-        process_pair(key.strip, value.strip.gsub(/\r?\n\s*/, ' '))
+        process_pair(key.strip, (value ? value.strip.gsub(/\r?\n\s*/, ' ') : nil))
```

Comment:
Should we return an empty string? It sounds like this code path would always return a string for well formed headers.